### ISSUE TITLE
Not sure if bug or not - huge project (~6000 classes) not seeing even one MA0053 (this is why I am testing this package :))

### ISSUE THREAD

**Issue by ShaharPrishMSFT** [meziantou/Meziantou.Analyzer#419]
**Context**
Rule identifier: _MA0053_
Target Framework: _.NET 6.0_
Language version: _10.0_

**Describe the bug**
I was expecting to see hundreds if not thousands of MA0053 on my solution (~21 projects with ~6000 classes) and am not seeing even one

**To Reproduce**
Code or steps to reproduce the behavior:

````c#
    internal class CorruptedAssetSnapshotException : Exception
    {
        public CorruptedAssetSnapshotException(OrgId orgId, Exception innerException)
            : base(string.Format($"{orgId} asset snapshot could not get loaded"), innerException)
        {
        }
    }
````

**Additional context**
Above is one obvious example that can be sealed. No warning is emitted.

**Comment by ShaharPrishMSFT** [comment]
Well.. That was quick...

I'll try out the editorconfig thingy. I dont recall ever configuring specific errors in such a way.

**Comment by meziantou** [comment]
Does enabling the options in the editorconfig file fix the issue?
